cypress/e2e/1-workflows.cy.ts

```diff
@@ -23,6 +23,7 @@ describe('Workflows', () => {
 	});
 
 	it('should create multiple new workflows using add workflow button', () => {
+		cy.viewport(1920, 1080);
```

Comment:
instead of expanding the viewport (which doesn't always seems to work), I think we should disable this botton in e2e tests, since there is no value having this in the tests.